Makefile

```diff
@@ -1,6 +1,9 @@
 SOURCES := $(shell find . 2>&1 | grep -E '.*\.(c|h|go)$$')
 
-.DEFAULT: ploop
+.DEFAULT: ploop.bin
 
-ploop: $(SOURCES)
-	go build -o ploop .
+ploop.bin: $(SOURCES)
+	go build -o ploop.bin .
+install: ploop.bin
+	cp ploop.sh /usr/libexec/kubernetes/kubelet-plugins/volume/exec/virtuozzo~ploop/ploop
```

Comment:
I'm constructive - I suggest to rewrite whole plugin in shell.
Necessity to copy 2 binaries is a significant drawback.